src/main/java/core/basesyntax/figures/Circle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.figures;
+
+public class Circle extends Figure {
+    private double radius;
+    private String color;
```

Comment:
Do not duplicate fields of parent class in child ones. It the reason we create parent class - to move there common fields.
Comment valid for all figures